mingw-w64-precice/PKGBUILD

```
@@ -26,7 +26,12 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-petsc-build"
 license=('LGPL3')
 url='https://www.precice.org'
 source=(https://github.com/precice/precice/archive/v${pkgver}.tar.gz)
-sha256sums=('57BAB08E8B986F5FAA364689D470940DBD9C138E5CFA7B861793E7DB56B89DA3')
+sha256sums=('762E603FBCAA96C4FB0B378B7CB6789D09DA0CF6193325603E5EEB13E4C7601C')
+
+prepare() {
+  cd "${_realname}-${pkgver}"
+  sed -i 's/BYPRODUCTS binprecice/#BYPRODUCTS binprecice/' CMakeLists.txt
```

This should work for packages until version 3

```suggestion
  sed -i 's/E create_symlink/E copy_if_different/' CMakeLists.txt
```